.buildbot.sh

```diff
@@ -39,10 +39,18 @@ cd ..
 rustup override set ykrustc-stage1
 
 # Test both workspaces using the compiler we just built.
+# We do this for both debug and release mode.
 export RUSTFLAGS="-C tracer=${CI_TRACER_KIND} -D warnings"
-cargo xtask test
-cargo xtask bench
-cargo xtask clean
+for mode in "" "--release"; do
+    cargo xtask test ${mode}
+    # check extra args are handled right.
```

Comment:
I'm not sure I understand this comment. The only possible arguments are the empty string (which is more a non-argument) and "--release", right?

Reply:
Like in line 47, you can pass extra arguments to cargo (e.g. test filtering) or to the test binary itself (e.g. `--nocapture`).

Reply:
Ah! So maybe we can be explicit here: "Check that extra arguments such as --nocapture work correctly"? Also, this only checks that they don't crash. Can we, for example, check the output with something like `cargo xtask test guard ${mode} -- --nocapture | grep <some value that should be produced>`? I appreciate that that might not be possible.

Reply:
I've done a more concise re-wording in 657a603, but I've refrained from using concrete opt-level numbers in the comment. I find that overly-specific comments beg to go stale. 

Hope that's OK?

Reply:
657a603 also greps for a test which shouldn't appear if the filter was correctly applied.

Reply:
Yes, I saw :)